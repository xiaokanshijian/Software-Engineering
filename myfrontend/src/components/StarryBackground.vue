<template>

  <div id="background-container">
    <vue-particles
        class="login-bg"
      color="#39AFFD"
      :particleOpacity="0.2"
      :particlesNumber="100"
      shapeType="circle"
      :particleSize="5"
      linesColor="#8DD1FE"
      :linesWidth="1"
      :lineLinked="true"
      :lineOpacity="0.4"
      :linesDistance="150"
      :moveSpeed="3"
      :hoverEffect="true"
      hoverMode="grab"
      :clickEffect="true"
      clickMode="push"
      >
      </vue-particles>
  </div>
</template>

<script>
export default {
  data(){
    return {

    }
  },
  components: {
  },
};
</script>


<style>
#particles-js{
  width: 99vw;
  height: 97vh;
  position: absolute;
  z-index: -1;
}
#background-container{
  width: 99vw;
  min-height: 100vh;
  position: absolute;
  z-index: -1;
  background: #051231;
}
</style>