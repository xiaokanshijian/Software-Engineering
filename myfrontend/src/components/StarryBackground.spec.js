import { shallowMount } from '@vue/test-utils';
import StarryBackground from './StarryBackground.vue';





describe('<StarryBackground/>', () => {
	const wrapper = shallowMount(StarryBackground);

	// 快照测试
	it('snapshot测试', () => {
		const wrapper2 = shallowMount(StarryBackground);
		expect(wrapper2.html()).toMatchSnapshot()
		wrapper2.destroy()
    })

	
  });

