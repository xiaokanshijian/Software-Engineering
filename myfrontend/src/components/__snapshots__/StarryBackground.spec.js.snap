// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StarryBackground/> snapshot测试 1`] = `
"<div id="background-container">
  <vue-particles color="#39AFFD" particleopacity="0.2" particlesnumber="100" shapetype="circle" particlesize="5" linescolor="#8DD1FE" lineswidth="1" linelinked="true" lineopacity="0.4" linesdistance="150" movespeed="3" hovereffect="true" hovermode="grab" clickeffect="true" clickmode="push" class="login-bg"></vue-particles>
</div>"
`;
